sklearn/tests/test_isotonic.py

```diff
@@ -511,6 +511,24 @@ def test_make_unique_dtype():
         assert_array_equal(x, [2, 3, 5])
 
 
+def test_make_unique_tol():
+    # Check that equality tolerance works
+    x = np.array([0, 1e-16, 1, 1+1e-14])
+    y = x.copy()
+    w = np.ones_like(x)
+    x, y, w = _make_unique(x, y, w)
+    assert_array_equal(x, [0, 1, 1+1e-14])
+
+
+def test_isotonic_inf():
+    # Check that inf values are not returned
```

Comment:
Please make it explicit that this is a non-regression test for a specific bug with the URL of the issue / PR.

Reply:
No problem, but is this a regression? The bug has not always been around?

Reply:
Nevermind, I just realised the difference between 'non-regression' and 'regression'